src/_modules/Commands.sol

```diff
@@ -147,97 +166,136 @@ library commands {
+    function run(string[11] memory inputs) internal returns (CommandResult memory) {
         return _toDynamic(inputs).run();
     }
 
-    function run(string[12] memory inputs) internal returns (bytes memory) {
+    function run(string[12] memory inputs) internal returns (CommandResult memory) {
         return _toDynamic(inputs).run();
     }
 
-    function run(string[13] memory inputs) internal returns (bytes memory) {
+    function run(string[13] memory inputs) internal returns (CommandResult memory) {
         return _toDynamic(inputs).run();
     }
 
-    function run(string[14] memory inputs) internal returns (bytes memory) {
+    function run(string[14] memory inputs) internal returns (CommandResult memory) {
         return _toDynamic(inputs).run();
     }
 
-    function run(string[15] memory inputs) internal returns (bytes memory) {
+    function run(string[15] memory inputs) internal returns (CommandResult memory) {
         return _toDynamic(inputs).run();
     }
 
-    function run(string[16] memory inputs) internal returns (bytes memory) {
+    function run(string[16] memory inputs) internal returns (CommandResult memory) {
         return _toDynamic(inputs).run();
     }
 
-    function run(string[17] memory inputs) internal returns (bytes memory) {
+    function run(string[17] memory inputs) internal returns (CommandResult memory) {
         return _toDynamic(inputs).run();
     }
 
-    function run(string[18] memory inputs) internal returns (bytes memory) {
+    function run(string[18] memory inputs) internal returns (CommandResult memory) {
         return _toDynamic(inputs).run();
     }
 
-    function run(string[19] memory inputs) internal returns (bytes memory) {
+    function run(string[19] memory inputs) internal returns (CommandResult memory) {
         return _toDynamic(inputs).run();
     }
 
-    function run(string[20] memory inputs) internal returns (bytes memory) {
+    function run(string[20] memory inputs) internal returns (CommandResult memory) {
         return _toDynamic(inputs).run();
     }
 
+    function isOk(CommandResult memory self) internal pure returns (bool) {
+        return self.exitCode == 0;
+    }
+
+    function isError(CommandResult memory self) internal pure returns (bool) {
+        return !self.isOk();
+    }
+
+    function getOutputOrRevert(CommandResult memory self, string memory customError)
+        internal
+        pure
+        returns (bytes memory)
+    {
+        if (self.isError()) {
+            revert(customError);
+        }
+
+        return self.stdout;
+    }
+
+    function getOutputOrRevert(CommandResult memory self) internal pure returns (bytes memory) {
+        string memory error;
+
+        if (self.isError()) {
+            error = string.concat("Failed to run command ", self.command.toString());
+
+            if (self.stderr.length > 0) {
+                error = string.concat(error, string.concat(": ", string(self.stderr)));
```

Comment:
string.concat receives multiple args, it can be `string.concat(error, ": ", string(self.stderr));`